utils/results/output/resultwriter.go

```diff
@@ -412,6 +415,10 @@ func PrintJasTable(tables formats.ResultsTables, entitledForJas bool, scanType j
 			return coreutils.PrintTable(tables.SastVulnerabilitiesTable, "Static Application Security Testing (SAST)",
 				"✨ No Static Application Security Testing vulnerabilities were found ✨", false)
 		}
+	case jasutils.MaliciousCode:
+		return coreutils.PrintTable(tables.MaliciousVulnerabilitiesTable, "Malicious Code Detection",
+			"✨ No Malicious Code vulnerabilities were found ✨", false)
```

Comment:
```suggestion
        if !violations {
		     return coreutils.PrintTable(tables.MaliciousVulnerabilitiesTable, "Malicious Code Detection",
			"✨ No Malicious Code vulnerabilities were found ✨", false)
	    }
```
we only want to print if not violations...

Reply:
ok